cpe_workspaceone/resources/cpe_workspaceone.rb

```diff
@@ -129,11 +164,13 @@ def macos_install
       pkg_url node['cpe_workspaceone']['pkg']['pkg_url'] if node['cpe_workspaceone']['pkg']['pkg_url']
       receipt node['cpe_workspaceone']['pkg']['receipt']
       version ws1_pkg_version
+      headers node['cpe_workspaceone']['pkg']['headers']
```

Comment:
many companies (like us) still use the legacy `cpe_remote`. I'm not sure if anything would blow up here, but I wonder if we can safely guard for this.

Reply:
I've gated this headers addition on headers being defined:

```ruby
unless node['cpe_workspaceone']['pkg']['headers'].nil?
        headers node['cpe_workspaceone']['pkg']['headers']
end
```